python/cac_tripplanner/cms/models.py

```diff
@@ -103,7 +103,7 @@ class ArticleTypes(object):
     modified = models.DateTimeField(auto_now=True)
     content_type = models.CharField(max_length=4, choices=ArticleTypes.CHOICES)
     wide_image = models.ImageField(upload_to=generate_filename, null=True,
-                                   help_text='The wide image. Will be displayed at 600x300.')
+                                   help_text='The wide image. Will be displayed at 1280x400.')
```

Comment:
This is lame, but Django will require a migration here even though the help_text isn't stored anywhere in the db.